torch/nn/functional.py

```diff
@@ -63,13 +72,23 @@
 Note:
     {cudnn_reproducibility_note}
 """.format(**reproducibility_notes, **tf32_notes) + r"""
+
+    Padding mode 'valid' is the same as no padding. 'same' pads the input so
```

Comment:
The start of this paragraph really jumps into to explain some padding options. Either start by explaining the padding parameter a little more, or include the "valid" vs. "same" padding effect in the args discussion below.